packages/fluentui/docs/src/examples/components/Chat/Content/ChatExampleActions.shorthand.tsx

```diff
@@ -34,7 +34,7 @@ const items: ShorthandCollection<ChatItemProps> = [
     ),
   },
   {
-    gutter: <Avatar image="https://fabricweb.azureedge.net/fabric-website/assets/images/avatar/small/ade.jpg" />,
+    gutter: <Avatar image="https://fabricweb.azureedge.net/fabric-website/assets/images/avatar/RobinCounts.jpg" />,
     message: <Chat.Message actionMenu={actionMenu} content="Hi" author="Jane Doe" timestamp="Yesterday, 10:15 PM" />,
```

Comment:
```suggestion
    message: <Chat.Message actionMenu={actionMenu} content="Hi" author="Robin Counts" timestamp="Yesterday, 10:15 PM" />,
```

Reply:
Please check `author` across `Chat` examples

Reply:
Ok, now it's either Cecil Folk or Robin Counts